site/content/user-guide/amqp-adapter.md

```diff
@@ -38,6 +35,34 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 ### SASL EXTERNAL Authentication
 TODO
 
+## AMQP Command-line Client
+For the purposes of demonstrating the usage of the AMQP adapter, the **Hono CLI Module** contains an AMQP command-line client for interacting with the AMQP adapter. This module does not activate the AMQP client by default and should be activated using the `--spring.profile.active` argument.
+
+The command-line client contains the following arguments (with default values):
+* `message.address`: The AMQP 1.0 message address (default: `telemetry/DEFAULT_TENANT/4711`)
+* `amqp.host`: The hostname that the AMQP adapter is running on (default: `localhost`)
+* `amqp.port`: The port that the adapter is listening for incoming connections (default: `4040`)
+* `username`: The username to authenticate to the adapter (default: `sensor1@DEFAULT_TENANT`)
+* `password`: The password to authenticate to the adapter (default: `hono-secret`)
+* `payload`: The message payload body (default: `'{"temp": 5}'`)
+* `--spring.profiles.active=amqp-adapter-cli`: Tells Hono CLI to activate the AMQP command-line client. This argument does not contain a default value and must be provided when running the client on the command-line.
```

Comment:
it does contain a default, but the default is to run the consumer ...

Reply:
yes, sure.. i think it will be better to remove the last sentence